pandas/tests/series/methods/test_convert_dtypes.py

```diff
@@ -226,3 +226,23 @@ def test_convert_bool_dtype(self):
         # GH32287
         df = pd.DataFrame({"A": pd.array([True])})
         tm.assert_frame_equal(df, df.convert_dtypes())
+
+    def test_convert_byte_string_dtype(self):
+        # https://github.com/pandas-dev/pandas/issues/43183 ->
+        # not recognising the byte_string dtype
+        byte_str = b"binary-string"
+
+        df = pd.DataFrame(
+            data={
+                "A": byte_str,
+            },
```

Comment:
The df creation should fit into one line?

Reply:
Done.